Source/CameraView/CameraView.swift

```diff
@@ -301,7 +301,7 @@ class CameraView: UIViewController, CLLocationManagerDelegate {
         self.focusImageView.transform = CGAffineTransformMakeScale(0.6, 0.6)
         }, completion: { _ in
           self.animationTimer = NSTimer.scheduledTimerWithTimeInterval(1, target: self,
-            selector: "timerDidFire", userInfo: nil, repeats: false)
+            selector: #selector(CameraView.timerDidFire), userInfo: nil, repeats: false)
```

Comment:
@RamonGilabert sadly, it has to be there, or "Call to method 'timerDidFire' in closure requires explicit 'self.' to make capture semantics explicit"